SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialPlayerSpawner.cpp

```diff
@@ -198,40 +198,58 @@ void USpatialPlayerSpawner::ReceivePlayerSpawnRequestOnServer(const Worker_Comma
 
 void USpatialPlayerSpawner::FindPlayerStartAndProcessPlayerSpawn(Schema_Object* SpawnPlayerRequest, const PhysicalWorkerName& ClientWorkerId)
 {
-	// If load-balancing is enabled AND the strategy dictates that another worker should have authority over
-	// the chosen PlayerStart THEN the spawn request is forwarded to that worker to prevent an initial player
-	// migration. Immediate player migrations can still happen if
-	// 1) the load-balancing strategy has different rules for PlayerStart Actors and Characters / Controllers /
+	// If the load balancing strategy dictates that this another worker should have authority the chosen PlayerStart THEN the spawn is handled locally,
+	// Else if the the PlayerStart is handled by another worker THEN forwarded the request to that worker to prevent an initial player migration
```

Comment:
```suggestion
	// Else if the the PlayerStart is handled by another worker THEN forward the request to that worker to prevent an initial player migration
```